x-pack/test/apm_api_integration/tests/transactions/latency.spec.ts

```diff
@@ -144,10 +147,50 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       });
 
       describe('99th percentile latency type', () => {
-        it('returns average duration and timeseries', async () => {
+        it('returns p99 duration and timeseries', async () => {
+          const response = await fetchLatencyCharts({
+            query: {
+              latencyAggregationType: LatencyAggregationType.p99,
+              useDurationSummary: false,
+            },
+          });
+
+          expect(response.status).to.be(200);
+          const latencyChartReturn = response.body as LatencyChartReturnType;
+
+          expect(latencyChartReturn.currentPeriod.overallAvgDuration).to.be(
+            expectedLatencyAvgValueMs
+          );
+          expect(latencyChartReturn.currentPeriod.latencyTimeseries.length).to.be.eql(15);
```

Comment:
Don't we need to assert any of the p99 values?

Reply:
Percentiles are a little harder due to the accuracy loss of histograms. We might need a different approach, but I'll follow up later if you don't mind.

Reply:
Sounds good to me 👍